sympy/physics/mechanics/functions.py

```diff
@@ -570,3 +570,14 @@ def flist_iter():
     unzip = lambda l: list(zip(*l)) if l[0] else [(), ()]
     vel_list, f_list = unzip(list(flist_iter()))
     return vel_list, f_list
+
+
+def convert_tuple_to_vector(frame, val_tuple):
```

Comment:
Please follow numpydoc string formatting and list the parameters, examples, etc.